src/lib/input/input.scss

```diff
@@ -138,6 +138,9 @@ select.mat-input-element {
   background-color: transparent;
   display: inline-flex;
   box-sizing: border-box;
+  padding-top: 20px;
```

Comment:
I just remembered, the `mat-form-field` uses a lot of `em` units so it can adjust to the current `font-size`. You might want to test a native select inside a form field with `font-size: 30px`. My guess is you're going to want to use `em` here too.